[2025-01-28T02:25:13.583+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T02:25:13.627+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2021-02-22T00:00:00+00:00 [queued]>
[2025-01-28T02:25:13.650+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2021-02-22T00:00:00+00:00 [queued]>
[2025-01-28T02:25:13.659+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-01-28T02:25:13.734+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): task1> on 2021-02-22 00:00:00+00:00
[2025-01-28T02:25:13.748+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1108) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-28T02:25:13.752+0000] {standard_task_runner.py:72} INFO - Started process 1121 to run task
[2025-01-28T02:25:13.747+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'task1', 'scheduled__2021-02-22T00:00:00+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmppzqohpal']
[2025-01-28T02:25:13.754+0000] {standard_task_runner.py:105} INFO - Job 56: Subtask task1
[2025-01-28T02:25:13.875+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.task1 scheduled__2021-02-22T00:00:00+00:00 [running]> on host 6d5b1ca1fccb
[2025-01-28T02:25:14.236+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2021-02-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-22T00:00:00+00:00'
[2025-01-28T02:25:14.241+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T02:25:14.295+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-28T02:25:14.298+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello World']
[2025-01-28T02:25:14.334+0000] {subprocess.py:99} INFO - Output:
[2025-01-28T02:25:14.349+0000] {subprocess.py:106} INFO - Hello World
[2025-01-28T02:25:14.356+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-28T02:25:14.490+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T02:25:14.491+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=task1, run_id=scheduled__2021-02-22T00:00:00+00:00, execution_date=20210222T000000, start_date=20250128T022513, end_date=20250128T022514
[2025-01-28T02:25:14.723+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T02:25:14.779+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T02:25:14.785+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
